### ISSUE TITLE
auth manager: login() for end user fails even if you include project and group 

### ISSUE THREAD

**Issue by mmrj** [forio/epicenter-js-libs#66]
user1 is an end user in several groups within this project.

```
var authMgr = new F.manager.AuthManager({ server: { host: 'api.forio.com' } });
authMgr.login({ 
           account: 'acme-simulations-0', 
           userName: 'user1',
           password: 'passw0rd',
           project: 'hello_world_python'
           group: 'js-libs-team3'
    	})
```
this call fails during  `https://api.forio.com/member/local/?userId=b1c19dda-2d2e-4777-ad5d-3929f17e86d3` because it has no Authorization header for the call to the Member API. 

debugged with @rvillarreal

**Comment by mmrj** [comment]
this is also causing failure in test/integration/multiplayer/test-script.js (for testing the Epicenter Channel Manager)

**Comment by mmrj** [comment]
to be verified, but marking as closed per @narenranjit 's comment that this is fixed by #75

**Comment by mmrj** [comment]
actually looks like this was closed by #70

**Comment by mmrj** [comment]
verified specifically that we're passing in the Authorization header for the calls to the Member API. 

(seeing some weirdness -- can log in if i don't pass either project or group, but problems when i pass in both. will log a separate issue once i've narrowed it down.)
